lib/ServiceManager.ts

```diff
@@ -93,7 +93,7 @@ export class ServiceManager implements ServiceManagerInterface {
 
   start() {
     if (this.started) {
-      this.stop();
```

Comment:
now I understand why you think it's a breaking change.

Reply:
agree this is a bug fix. maybe we should clarify the behavior in README. calling start does not RESTART the service. If you want to do that, you should call stop() then start()